From 7ee7e59e858c7f1901c6879e39b30480c42ef126 Mon Sep 17 00:00:00 2001
From: popcornmix <popcornmix@gmail.com>
Date: Mon, 29 Apr 2013 22:50:08 +0100
Subject: [PATCH] [rbp] Remove unnecessary gl ifdef

I'm not sure why this was originally added.
I couldn't think of a reason why the Pi would want things done differently here,
so I tried without the ifdef. I can't see any difference in behaviour,
so I think we're better off removing it.
---
 xbmc/rendering/gles/RenderSystemGLES.cpp | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/xbmc/rendering/gles/RenderSystemGLES.cpp b/xbmc/rendering/gles/RenderSystemGLES.cpp
index e7795fb..472f441 100644
--- a/xbmc/rendering/gles/RenderSystemGLES.cpp
+++ b/xbmc/rendering/gles/RenderSystemGLES.cpp
@@ -138,11 +138,7 @@ bool CRenderSystemGLES::ResetRenderSystem(int width, int height, bool fullScreen
   g_matrices.MatrixMode(MM_PROJECTION);
   g_matrices.LoadIdentity();
 
-#ifdef TARGET_RASPBERRY_PI
-  g_matrices.Ortho(0.0f, width-1, height-1, 0.0f, +1.0f, 1.0f);
-#else
   g_matrices.Ortho(0.0f, width-1, height-1, 0.0f, -1.0f, 1.0f);
-#endif
 
   g_matrices.MatrixMode(MM_MODELVIEW);
   g_matrices.LoadIdentity();
-- 
1.8.1.6

